hbw/app/services/hbw/task_notifier.rb

```diff
@@ -0,0 +1,28 @@
+module HBW
+  class TaskNotifier
+    class << self
+      def call(widget, task_id, event_name, users = [])
+        cache_key = "#{task_id}_#{DateTime.current.strftime('%Q')}_#{SecureRandom.hex(10)}"
```

Comment:
Are you sure we need `DateTime.current` from `ActiveSupport`? I think we can use native `DateTime.now` here.